keras_nlp/src/tests/test_case.py

```diff
@@ -419,20 +420,22 @@ def run_backbone_test(
                 self.assertEqual(output[key].shape, expected_output_shape[key])
         else:
             self.assertEqual(output.shape, expected_output_shape)
-
-        # Check we can embed tokens eagerly.
-        output = backbone.token_embedding(ops.zeros((2, 3), dtype="int32"))
-
-        # Check variable length sequences.
-        if variable_length_data is None:
-            # If no variable length data passed, assume the second axis of all
-            # inputs is our sequence axis and create it ourselves.
-            variable_length_data = [
-                tree.map_structure(lambda x: x[:, :seq_length, ...], input_data)
-                for seq_length in (2, 3, 4)
-            ]
-        for batch in variable_length_data:
-            backbone(batch)
+        if not is_cv_backbone:
```

Comment:
what if we just remove this option for now, and make this if...

`if backbone.token_embedding is not None:`

That seems a fine proxy for "is this an NLP model" for now.